std/range/interfaces.d

```diff
@@ -208,6 +208,13 @@ interface RandomAccessInfinite(E) : ForwardRange!E {
 interface InputAssignable(E) : InputRange!E {
     ///
     @property void front(E newVal);
+
+    @property E front();
```

Comment:
Yes it is,  _all_ public functions should be exposed on the documentation - this includes range functions as well ;-)
(the reason for this is that it's an easy way to show what capabilities are provided by a range.